src/en/controllers.md

```diff
@@ -0,0 +1,40 @@
+Title: Juju Controllers
+TODO: Review again soon (created: March 2016)
+
+
+# Controllers
+
+A Juju *controller* is a machine that gets provisioned when the initial cloud
+provider *model* is created. The controller is therefore also called the
+*controller model* due to it being associated with that initial model.  It
+describes this model whose primary purpose is to run and manage the Juju API
+servers and underlying database.
+
+Once a controller model has been created you can define multiple models within
+that environment. See [Models](./models.html).
+
+When a controller is intended to be used by multiple people, it is recommended
+that Juju's [multi-user functionality](./juju-multiuser-environments.html) be
+leveraged.
+
+It is not possible to create a second controller model for the same cloud
```

Comment:
this is literally true but misleading. You can create a cloud (e.g. rax2) and then have a new controller in that